docs/cado-response/deploy/gcp/gcp-settings.md

```diff
@@ -26,31 +26,95 @@ In order to set up the project for Cado collection you need to
 
 ***Note - if you are importing from more than one Google project you will need to designate one of those projects as the Primary GCP Project and [create a bucket in that project](./gcp-settings.md#defining-a-bucket-for-the-primary-gcp-project) to enable collection across the multiple Google projects***
 
-### Creating a Cado Role
-To appropriately scope a service account for Cado to operate, creating a custom GCP role allows specifying individual permissions. To achieve this, navigate to the **Role** section un the **IAM and Admin** tab. Create a custom role and add the following permissions:
+### The Cado Role
+To appropriately scope a service account for Cado to operate, the Terraform script will have created a custom GCP role, prefixed with **myCadoResponseRole**. To achieve this, the role will have the following permissions, which can be checked by navigating to the **Role** section un the **IAM and Admin** tab.
 ```
-  cloudbuild.builds.create
-  cloudbuild.builds.get
-  compute.disks.get
-  compute.disks.useReadOnly
-  compute.globalOperations.get
-  compute.images.create
-  compute.instances.get
-  compute.instances.list
-  container.clusters.get
-  container.clusters.list
-  container.pods.exec
-  container.pods.get
-  container.pods.list
-  iam.serviceAccounts.getAccessToken
-  iam.serviceAccounts.implicitDelegation
-  resourcemanager.projects.get
-  storage.buckets.get
-  storage.buckets.list
-  storage.objects.get
-  storage.objects.list
+    // Instance Acquisition
```

Comment:
We could reference a link to https://github.com/cado-security/Deployment-Templates/blob/main/gcp/modules/iam/main.tf which will always have the latest